apps/demo-game/src/pages/index.tsx

```diff
@@ -1,67 +1,183 @@
 import {
+  Layout,
   PlayerDisplay,
   ProbabilityChart,
   StorageOverview,
   Timeline,
   TimelineEntry,
+  TradingForm,
 } from '@gbl-uzh/ui'
+
 import { useRouter } from 'next/router'
 
 export default function Home() {
   const router = useRouter()
   return (
-    <div className="font-source-sans p-4">
-      hello world
-      {/* <Button /> */}
-      {/* <Dice dice1={'dice1'} dice2={'dice2'} /> */}
-      {/* <Progress value={50} max={100} formatter={(val) => `${val}XP`} /> */}
-      <div className="flex w-1/3 flex-col">
-        <PlayerDisplay
-          name={'playerName'}
-          avatar={'avatar_placeholder'}
-          color={'Red'}
-          achievements={[]}
-          location={'ZH'}
-          xp={20}
-          xpToNext={50}
-          level={1}
-          onClick={() => {
-            router.replace('/play/welcome')
+    <div>
+      <Layout>
+        <div className="w-full rounded bg-black"></div>
+      </Layout>
+      <div className="font-source-sans p-4">
+        hello world
+        {/* <Button /> */}
+        {/* <Dice dice1={'dice1'} dice2={'dice2'} /> */}
+        {/* <Progress value={50} max={100} formatter={(val) => `${val}XP`} /> */}
+        <div className="flex w-1/3 flex-col">
+          <PlayerDisplay
+            name={'playerName'}
+            imgPathAvatar={'/avatars/avatar_placeholder.png'}
+            color={'Red'}
+            achievements={[]}
+            imgPathLocation={'/locations/ZH.svg'}
+            location={'ZH'}
+            xp={20}
+            xpMax={50}
+            level={1}
+            onClick={() => {
+              router.replace('/play/welcome')
+            }}
+          />
+          <StorageOverview
+            storageTotal={3}
+            storageUsed={1}
+            imgPathEmpty={'/avatars/cocoa_0.png'}
+            imgPathFull={'/avatars/cocoa_3.png'}
+          />
+        </div>
+        <ProbabilityChart trendE={5} trendGap={8} totalEyes={'12'} />
+        <TimelineEntry
+          periodIx={1}
+          segmentIx={1}
+          isCurrentEntry={true}
```

Comment:
```suggestion
          isCurrentEntry
```

this is cleaner, set the default values to false, and only if they are added like this, they are true. allows to instantly see that they are binary toggles, and isPastEntry={false} does not need to be listed as a prop

Reply:
Fair enought, just looked it up, it belongs indeed to best practice, thanks!

Wrt isPastEntry: I thought that as well, but I think you intended to show past entries and future ones differently, that is we either need to tell if the other entries are past or future ones. the current one is highlighted, the future ones are black and the past are gray. 

What we could do is to provide an enum (possible states: past, current, future) instead of two booleans - this might be cleaner - what do you think?